.pre-commit-config.yaml

```diff
@@ -1,7 +1,19 @@
 default_language_version:
-  node: 18.15.0
+  node: system
 
 repos:
+  # EOF and whitespace checker
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: end-of-file-fixer
+        exclude: |
+            (?x)^(
+                site/.*|
+                tests/assets/.*
+            )$
+      - id: trailing-whitespace
```

Comment:
AFAIK, trailing-whitespace can be covered by 'git diff --check' which is included in this config from the repo 'https://github.com/jumanjihouse/pre-commit-hooks' below. So, I think if it is for doing same thing, it's better to omit this hook.

Reply:
Removed